<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>AddonsInstaller</name>
    <message>
        <location filename="package_details.py" line="112"/>
        <source>Version {version} installed on {date}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="117"/>
        <source>Version {version} installed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="121"/>
        <source>Installed on {date}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="567"/>
        <source>Installed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="96"/>
        <source>Version</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="103"/>
        <source>Unknown version (no package.xml file found)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="108"/>
        <source>installed on</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="118"/>
        <source>installed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="126"/>
        <source>Update available to version</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="144"/>
        <source>This is the latest version available</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="151"/>
        <source>Updated, please restart FreeCAD to use</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="159"/>
        <source>Update check in progress</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="163"/>
        <source>Automatic update checks disabled</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="172"/>
        <source>Installation location</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="240"/>
        <source>WARNING: This addon is obsolete</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="250"/>
        <source>WARNING: This addon is Python 2 Only</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="312"/>
        <source>Fetching README.md from package repository</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="354"/>
        <source>WARNING: This addon requires FreeCAD </source>
        <translation type="unfinished"></translation>
    </message>
    <message numerus="yes">
        <location filename="package_details.py" line="431"/>
        <source>Delete cached version of this README and re-download</source>
        <translation type="unfinished">
            <numerusform></numerusform>
        </translation>
    </message>
    <message>
        <location filename="package_details.py" line="563"/>
        <source>Could not load README data from URL {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="577"/>
        <source>Success</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="600"/>
        <source>Changed to git ref &apos;{}&apos; -- please restart to use Addon.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="636"/>
        <source>Page JavaScript reported</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="407"/>
        <source>Install</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="410"/>
        <source>Uninstall</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="413"/>
        <source>Update</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="416"/>
        <source>Check for Update</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="419"/>
        <source>Run Macro</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="786"/>
        <source>Change Branch</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_details.py" line="424"/>
        <source>Return to package list</source>
        <translation type="unfinished"></translation>
    </message>
    <message numerus="yes">
        <location filename="package_details.py" line="800"/>
        <source>QtWebEngine Python bindings not installed -- using fallback README display. See Report View for details and installation instructions.</source>
        <translation type="unfinished">
            <numerusform></numerusform>
        </translation>
    </message>
    <message>
        <location filename="package_list.py" line="119"/>
        <source>Filter is valid</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="125"/>
        <source>Filter regular expression is invalid</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="207"/>
        <source>Click for details about package {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="211"/>
        <source>Click for details about workbench {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="215"/>
        <source>Click for details about macro {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="326"/>
        <source>Maintainer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="330"/>
        <source>Maintainers:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="364"/>
        <source>Tags</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="371"/>
        <source>updated</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="376"/>
        <source>Updated</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="382"/>
        <source>Author</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="404"/>
        <source>Up-to-date</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="586"/>
        <source>Update available</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="414"/>
        <source>Pending restart</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="180"/>
        <source>Click for details about package</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="185"/>
        <source>Click for details about workbench</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="190"/>
        <source>Click for details about macro</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="372"/>
        <source>Installed version</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="380"/>
        <source>Unknown version</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="383"/>
        <source>Installed on</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="394"/>
        <source>Available version</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="566"/>
        <source>Show Addons containing:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="569"/>
        <source>All</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="572"/>
        <source>Workbenches</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="575"/>
        <source>Macros</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="578"/>
        <source>Preference Packs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="692"/>
        <source>Status:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="695"/>
        <source>Any</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="702"/>
        <source>Not installed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="582"/>
        <source>Filter</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="package_list.py" line="585"/>
        <source>OK</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="115"/>
        <source>Checking network connection...
</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="167"/>
        <source>Unable to download addon list.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="172"/>
        <source>Workbenches list was updated.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="261"/>
        <source>Failed loading</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="296"/>
        <source>List of macros successfully retrieved.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="310"/>
        <source>GitPython not installed! Cannot retrieve macros from git</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="319"/>
        <source>Something went wrong with the Git Macro Retrieval, possibly the Git executable is not in the path</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="343"/>
        <source>Appears to be an issue connecting to the Wiki, therefore cannot retrieve Wiki macro list at this time</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="367"/>
        <source>Unable to fetch git updates for workbench</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="391"/>
        <source>Retrieving info from</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="419"/>
        <source>Failed to read metadata from</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="426"/>
        <source>git fetch failed for {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="429"/>
        <source>git fetch failed for {wb.name}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="433"/>
        <source>Raw markdown displayed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="435"/>
        <source>Python Markdown library is missing.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="439"/>
        <source>git pull failed for {wb.name}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="514"/>
        <source>Done locating macros.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="521"/>
        <source>This addon is already installed.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="533"/>
        <source>An update is available for this addon.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="547"/>
        <source>This addon is marked as obsolete</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="551"/>
        <source>This usually means it is no longer maintained, and some more advanced addon in this list provides the same functionality.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="559"/>
        <source>This addon is marked as Python 2 Only</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="564"/>
        <source>This workbench may no longer be maintained and installing it on a Python 3 system will more than likely result in errors at startup or while in use.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="608"/>
        <source>An error occurred fetching macros from GitHub</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="651"/>
        <source>Retrieving description...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="665"/>
        <source>This macro is already installed.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="697"/>
        <source>Caching macro code...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="700"/>
        <source>GitPython not found. Using standard download instead.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="705"/>
        <source>Your version of python doesn&apos;t appear to support ZIP files. Unable to proceed.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="727"/>
        <source>User requested updating a Python 2 workbench on a system running Python 3 - </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="738"/>
        <source>Outdated GitPython detected, consider upgrading with pip.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="763"/>
        <source>Workbench successfully updated. Please restart FreeCAD to apply the changes.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="771"/>
        <source>User requested installing a Python 2 workbench on a system running Python 3 - </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="786"/>
        <source>Workbench successfully installed. Please restart FreeCAD to apply the changes.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="799"/>
        <source>A macro has been installed and is available under Macro -&gt; Macros menu</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="824"/>
        <source>Retrieving info from {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="835"/>
        <source>Missing workbench</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="844"/>
        <source>Missing python module</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="854"/>
        <source>Missing optional python module (doesn&apos;t prevent installing)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="857"/>
        <source>Some errors were found that prevent to install this workbench</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="859"/>
        <source>Please install the missing components first.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="873"/>
        <source>Error: Unable to locate zip from</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="880"/>
        <source>Error: Unable to download</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="893"/>
        <source>Successfully installed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="915"/>
        <source>Retrieving macro description...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="653"/>
        <source>Retrieving info from git</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="656"/>
        <source>Retrieving info from wiki</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1075"/>
        <source>Error updating module </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1078"/>
        <source>Please fix manually</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1220"/>
        <source>Error: Unable to locate ZIP from</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1403"/>
        <source>Retrieving package metadata...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1405"/>
        <source>Failed to execute pip. Returned error was:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1611"/>
        <source>Downloaded package.xml for {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1641"/>
        <source>Downloaded metadata.txt for {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_workers.py" line="1715"/>
        <source>Downloaded icon for {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_utilities.py" line="332"/>
        <source>Failed to repair missing .git directory</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_utilities.py" line="335"/>
        <source>Repository URL</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_utilities.py" line="338"/>
        <source>Clone directory</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_metadata.py" line="72"/>
        <source>Error with encrypted connection</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_metadata.py" line="135"/>
        <source>Failed to connect to</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_metadata.py" line="220"/>
        <source>Failed to connect to URL</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="change_branch.py" line="101"/>
        <source>DANGER: Developer feature</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="change_branch.py" line="114"/>
        <source>There are local changes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="change_branch.py" line="192"/>
        <source>Branch</source>
        <comment>git terminology</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="change_branch.py" line="178"/>
        <source>Tag</source>
        <comment>git terminology</comment>
        <translation type="unfinished"></translation>
    </message>
    <message numerus="yes">
        <location filename="change_branch.py" line="261"/>
        <source>Kind</source>
        <comment>Table header for git ref type (e.g. either Tag or Branch)</comment>
        <translation type="unfinished">
            <numerusform></numerusform>
        </translation>
    </message>
    <message>
        <location filename="change_branch.py" line="265"/>
        <source>Local name</source>
        <comment>Table header for git ref name</comment>
        <translation type="unfinished"></translation>
    </message>
    <message numerus="yes">
        <location filename="change_branch.py" line="271"/>
        <source>Tracking</source>
        <comment>Table header for git remote tracking branch name name</comment>
        <translation type="unfinished">
            <numerusform></numerusform>
        </translation>
    </message>
    <message numerus="yes">
        <location filename="change_branch.py" line="277"/>
        <source>Local updated</source>
        <comment>Table header for git update time of local branch</comment>
        <translation type="unfinished">
            <numerusform></numerusform>
        </translation>
    </message>
    <message numerus="yes">
        <location filename="change_branch.py" line="283"/>
        <source>Remote updated</source>
        <comment>Table header for git update time of remote branch</comment>
        <translation type="unfinished">
            <numerusform></numerusform>
        </translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="191"/>
        <source>Checking connection</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="192"/>
        <source>Checking for connection to GitHub...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="213"/>
        <source>Connection failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="240"/>
        <source>Missing dependency</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="324"/>
        <source>Starting up...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="258"/>
        <source>Loading addon information</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="199"/>
        <source>Addon manager</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="209"/>
        <source>Restart now</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="210"/>
        <source>Restart later</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1248"/>
        <source>Refresh local cache</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="659"/>
        <source>Updating cache...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="86"/>
        <source>The addons that can be installed here are not officially part of FreeCAD, and are not reviewed by the FreeCAD team. Make sure you know what you are installing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="203"/>
        <source>You must restart FreeCAD for changes to take effect.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="204"/>
        <source>You must restart FreeCAD for changes to take effect. Press Ok to restart FreeCAD now, or Cancel to restart later.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="241"/>
        <source>Could not import QtNetwork -- see Report View for details. Addon Manager unavailable.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="243"/>
        <source>Checking for updates...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="262"/>
        <source>Apply</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="263"/>
        <source>update(s)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="266"/>
        <source>No update available</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="433"/>
        <source>Macro successfully installed. The macro is now available from the Macros dialog.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="435"/>
        <source>Unable to install</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="494"/>
        <source>Addon successfully removed. Please restart FreeCAD</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="496"/>
        <source>Unable to remove this addon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="502"/>
        <source>Macro successfully removed.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="534"/>
        <source>Addon Manager: Skipping update check because AutoCheck user preference is False</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="542"/>
        <source>Restart required</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="773"/>
        <source>Apply {} update(s)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="577"/>
        <source>No updates available</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="774"/>
        <source>All packages updates failed. Packages:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="786"/>
        <source>Failed packages:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="915"/>
        <source>Too many to list</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="893"/>
        <source>Missing Requirement</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="964"/>
        <source>Installing dependencies</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1061"/>
        <source>Cannot execute Python</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1081"/>
        <source>Cannot execute pip</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1102"/>
        <source>Package installation failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1105"/>
        <source>See Report View for detailed failure log.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="703"/>
        <source>Installation of macro failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message numerus="yes">
        <location filename="AddonManager.py" line="1280"/>
        <source>{} total, see Report view for list</source>
        <comment>Describes the number of updates that were completed (&apos;{}&apos; is replaced by the number of updates)</comment>
        <translation type="unfinished">
            <numerusform></numerusform>
        </translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1178"/>
        <source>Succeeded</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1513"/>
        <source>Execution of Addon&apos;s uninstall.py script failed. Proceeding with uninstall...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1159"/>
        <source>All packages updates failed:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1183"/>
        <source>Failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="800"/>
        <source>Update report</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="850"/>
        <source>Installation succeeded</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="866"/>
        <source>Installation failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1457"/>
        <source>Confirm remove</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="1460"/>
        <source>Are you sure you want to uninstall this Addon?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="504"/>
        <source>Macro could not be removed.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_macro.py" line="157"/>
        <source>Unable to fetch the code of this macro.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="addonmanager_macro.py" line="164"/>
        <source>Unable to retrieve a description for this macro.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>Std_AddonMgr</name>
    <message>
        <location filename="AddonManager.py" line="68"/>
        <source>&amp;Addon manager</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="AddonManager.py" line="69"/>
        <source>Manage external workbenches and macros</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>AddonInstaller</name>
    <message>
        <location filename="addonmanager_workers.py" line="535"/>
        <source>Installed location</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>Dialog</name>
    <message>
        <location filename="../../AddonManager.ui" line="14"/>
        <source>Addon Manager</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="37"/>
        <source>Workbenches</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="47"/>
        <source>Macros</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="56"/>
        <source>Executes the selected macro, if installed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="59"/>
        <source>Execute</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="89"/>
        <source>Sets configuration options for the Addon Manager</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="92"/>
        <source>Configure...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="113"/>
        <source>Downloading info...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="127"/>
        <source>Uninstalls a selected macro or workbench</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="130"/>
        <source>Uninstall selected</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="137"/>
        <source>Installs or updates the selected macro or workbench</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="140"/>
        <source>Install/update selected</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="147"/>
        <source>Download and apply all available updates</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="150"/>
        <source>Update all</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="157"/>
        <source>Close the Addon Manager</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManager.ui" line="160"/>
        <source>Close</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="14"/>
        <source>Addon manager options</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="20"/>
        <source>If this option is selected, when launching the Addon Manager,
installed addons will be checked for available updates
(this requires the GitPython package installed on your system)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="25"/>
        <source>Automatically check for updates at start (requires GitPython)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="35"/>
        <source>Custom repositories (one per line):</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="42"/>
        <source>You can use this window to specify additional addon repositories
to be scanned for available addons</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="57"/>
        <source>Proxy </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="64"/>
        <source>No proxy</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="71"/>
        <source>User system proxy</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../../AddonManagerOptions.ui" line="78"/>
        <source>User defined proxy :</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
